chains/musicoin.json

```diff
@@ -0,0 +1,168 @@
+		"enode://3f1d12044546b76342d59d4a05532c14b85aa669704bfe1f864fe079415aa2c02d743e03218e57a33fb94523adb54032871a6c51b2cc5514cb7c7e35b3ed0a99@13.93.211.84:30303",
+		"enode://78de8a0916848093c73790ead81d1928bec737d565119932b98c6b100d944b7a95e94f847f689fc723399d2e31129d182f7ef3863f2b4c820abbf3ab2722344d@191.235.84.50:30303",
+		"enode://158f8aab45f6d19c6cbf4a089c2670541a8da11978a2f90dbf6a502a4a3bab80d288afdbeb7ec0ef6d92de563767f3b1ea9e8e334ca711e9f8e2df5a0385e8e6@13.75.154.138:30303",
+		"enode://1118980bf48b0a3640bdba04e0fe78b1add18e1cd99bf22d53daac1fd9972ad650df52176e7c7d89d1114cfef2bc23a2959aa54998a46afcf7d91809f0855082@52.74.57.123:30303",
+		"enode://979b7fa28feeb35a4741660a16076f1943202cb72b6af70d327f053e248bab9ba81760f39d0701ef1d8f89cc1fbd2cacba0710a12cd5314d5e0c9021aa3637f9@5.1.83.226:30303",
+		"enode://d302f52c8789ad87ee528f1431a67f1aa646c9bec17babb4665dfb3d61de5b9119a70aa77b2147a5f28854092ba09769323c1c552a6ac6f6a34cbcf767e2d2fe@158.69.248.48:30303",
+		"enode://c72564bce8331ae298fb8ece113a456e3927d7e5989c2be3e445678b3600579f722410ef9bbfe339335d676af77343cb21b5b1703b7bebc32be85fce937a2220@191.252.185.71:30303"
+	],
+	"accounts":{
+		"0000000000000000000000000000000000000001":{
+			"balance":"1",
+			"builtin":{
+				"name":"ecrecover",
+				"pricing":{
+					"linear":{
+						"base":3000,
+						"word":0
+					}
+				}
+			}
+		},
+		"0000000000000000000000000000000000000002":{
+			"balance":"1",
+			"builtin":{
+				"name":"sha256",
+				"pricing":{
+					"linear":{
+						"base":60,
+						"word":12
+					}
+				}
+			}
+		},
+		"0000000000000000000000000000000000000003":{
+			"balance":"1",
+			"builtin":{
+				"name":"ripemd160",
+				"pricing":{
+					"linear":{
+						"base":600,
+						"word":120
+					}
+				}
+			}
+		},
+		"0000000000000000000000000000000000000004":{
+			"balance":"1",
+			"builtin":{
+				"name":"identity",
+				"pricing":{
+					"linear":{
+						"base":15,
+						"word":3
+					}
+				}
+			}
+		},
+		"0000000000000000000000000000000000000005":{
+			"builtin":{
+				"name":"modexp",
+				"activate_at":"0x21e88e",
+				"pricing":{
+					"modexp":{
+						"divisor":20
+					}
+				}
+			}
+		},
+		"0000000000000000000000000000000000000006":{
+			"builtin":{
+				"name":"alt_bn128_add",
+				"activate_at":"0x21e88e",
+				"pricing":{
+					"linear":{
+						"base":500,
+						"word":0
+					}
+				}
+			}
+		},
+		"0000000000000000000000000000000000000007":{
+			"builtin":{
+				"name":"alt_bn128_mul",
+				"activate_at":"0x21e88e",
+				"pricing":{
+					"linear":{
+						"base":40000,
+						"word":0
+					}
+				}
+			}
+		},
+		"0000000000000000000000000000000000000008":{
+			"builtin":{
+				"name":"alt_bn128_pairing",
+				"activate_at":"0x21e88e",
+				"pricing":{
+					"alt_bn128_pairing":{
+						"base":100000,
+						"pair":80000
+					}
+				}
+			}
+		}
+	}
+}
```

Comment:
Here's a blog post describing why GitHub warns about missing newlines at the end of files: https://robots.thoughtbot.com/no-newline-at-end-of-file